src/internal/xml-parser.ts

```diff
@@ -394,3 +397,145 @@ export function parseBucketVersioningConfig(xml: string) {
   const xmlObj = parseXml(xml)
   return xmlObj.VersioningConfiguration
 }
+
+// Used only in selectObectContent API.
```

Comment:
Looks like there's a very minor typo here:

```suggestion
// Used only in selectObjectContent API.
```